src/core/server/ui_settings/ui_settings_client.ts

```diff
@@ -120,7 +120,7 @@ export class UiSettingsClient implements IUiSettingsClient {
   }
 
   getOverrideOrDefault(key: string): unknown {
-    return this.isOverridden(key) ? this.overrides[key].value : this.defaults[key]?.value;
+    return this.isOverridden(key) ? this.overrides[key] : this.defaults[key]?.value;
```

Comment:
This was the reason for the original revert. Can you validate this? I think this is right, but just to be sure.

Reply:
Did. This is the correct way of reading `overrides`. In fact, i remember when I added `.value`... hadn't slept for a day or two and my eyes were blurry.